src/AccountData.js

```diff
@@ -21,14 +17,24 @@ class AccountData extends Component {
 	}
 
 	render() {
+		// Use lowercase units. See https://web3js.readthedocs.io/en/1.0/web3-utils.html#fromwei.
+		const units = this.props.units ? this.props.units.toLowerCase() : 'wei'
+		var balance = this.context.drizzle.web3.utils.fromWei(this.state.balance, units)
```

Comment:
`this.context` should be `this.props`

Reply:
Turns out there was another issue hidden here. The default value of `null` for the balance was throwing an error in web3.utils.fromWei; I've gone ahead and changed the default to 0, which fixes this.